tests/test_api.py

```diff
@@ -9,7 +9,7 @@ def test_deprecate_region_iteration(mock_boto, mock_get_all_regions):
     regions = ["region-1", "region-2", "region-3"]
     mock_get_all_regions.return_value = regions
 
-    config = configmodels.ConfigModel(**{"images": {}})
+    config = configmodels.ConfigModel(**{"images": {}, "options": {}})
```

Comment:
can we add tests to ensure options are handled correctly in `_get_images`?

Reply:
I opted not to as they're just passed transparently from config to boto3. It seems like any testing I do would either be testing pydantic or boto3 but nothing on our end

Reply:
it would be worthwhile to explicitly test the supported options, even if we aren't testing _get_images directly. my preference here is to ensure we have tests that catch inadvertent impl changes.